packages/action-group/stories/action-group-sizes.stories.ts

```diff
@@ -0,0 +1,173 @@
+        justified: false,
+        quiet: false,
+        vertical: false,
+        size: 'm',
+    },
+    argTypes: {
+        compact: {
+            name: 'compact',
+            description:
+                'Visually joins the buttons together to clarify their relationship to one another.',
+            type: { name: 'boolean', required: false },
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: false },
+            },
+            control: {
+                type: 'boolean',
+            },
+        },
+        emphasized: {
+            name: 'emphasized',
+            type: { name: 'boolean', required: false },
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: false },
+            },
+            control: {
+                type: 'boolean',
+            },
+        },
+        justified: {
+            name: 'justified',
+            description:
+                'Aligns the action group items to use all the available space on that line.',
+            type: { name: 'boolean', required: false },
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: false },
+            },
+            control: {
+                type: 'boolean',
+            },
+        },
+        quiet: {
+            name: 'quiet',
+            type: { name: 'boolean', required: false },
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: false },
+            },
+            control: {
+                type: 'boolean',
+            },
+        },
+        vertical: {
+            name: 'vertical',
+            description: 'Changes the orientation of the action group.',
+            type: { name: 'boolean', required: false },
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: false },
+            },
+            control: {
+                type: 'boolean',
+            },
+        },
+        size: {
+            name: 'size',
+            description: 'The size at which to display the action group.',
+            type: { name: 'string', required: true },
+            table: {
+                type: { summary: '"s" | "m" | "l" | "xl"' },
+                defaultValue: { summary: 'm' },
+            },
+            control: {
+                type: 'select',
+                options: ['s', 'm', 'l', 'xl'],
+            },
+        },
+    },
+};
+
+interface Properties {
+    compact?: boolean;
+    emphasized?: boolean;
+    justified?: boolean;
+    quiet?: boolean;
+    vertical?: boolean;
+    size?: 's' | 'm' | 'l' | 'xl';
+    [prop: string]: unknown;
+}
+
+function renderButtons(args: Properties): TemplateResult {
+    return html`
+        <sp-action-group ${spreadProps(args)}>
+            <sp-action-button size=${args.size}>Button 1</sp-action-button>
```

Comment:
I think you're passing the `size` internally, so they don't need to be on each Action Button anymore, right?